### ISSUE TITLE
Search bar in 'Admin' tab of publisher page is not prefilled with 'publisher:'

### ISSUE THREAD

**Issue by szakarias** [dart-lang/pub-dev#7317]
It is prefilled in the 'Packages' tab of the publisher page.

**Comment by isoos** [comment]
It is also missing from the activity log page.
Another inconsistency: if the publisher has unlisted packages, the navigation tab is missing from the admin pages.
